playbook/app/pb_kits/playbook/pb_copy_button/docs/_copy_button_hook.md

```diff
@@ -0,0 +1 @@
+We provide a `usePBCopy` hook that you can import to your project. This hook will return a function that you can call to copy the text to the clipboard.
```

Comment:
This is great! Can we add copy for the connection the 'from' and the id on the text we're copying? Just so it's clearer hoe to use this!

Do you think a click to copy may make more sense as a doc example instead of a hover to copy?

Finally, is there a way to get some feedback that shows that text has been copied? For the default version we have the tooltip so it's clear it's been copied, any way to do that with the hook as well? Even if it's just something that can be done in the doc?